app/src/app/api/v1/.well-known/mcp-server/route.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @swagger
+ * /api/v1/.well-known/mcp-server:
+ *   get:
```

Comment:
Would be good to move this before the method definition as it documents the GET method (and there can be multiple in case of multiple HTTP method for the same path).